server/index.js

```diff
@@ -18,63 +19,93 @@ app.use('*.js', (req, res, next) => {
 app.use(express.static('./client/dist'));
 app.use(express.json());
 
-/* ******** Personnel Routes ***************** */
+/* ******** Get Routes ***************** */
+
+app.get('/api/movies', async (req, res) => {
+  console.log('Received GET at /api/movies');
+  // find one movie object,
+  // find related personnel,
+  // populate movie object with personnel list.
+  let movie;
+  await db.getMovieById(Number(req.query.id))
+    .then((results) => {
+      movie = results[0];
+    })
+    .catch(err => console.error(err));
+
+  await db.getRelatedPersonnel(Number(req.query.id))
+    .then((results) => {
+      movie.personnel = results.rows;
+      res.send(movie);
+    })
+    .catch(err => console.error(err));
+});
 
-app.get('/api/personnel', (req, res) => {
-  // find related personnel for a given movie id
-  db.getRelatedPersonnel(Number(req.query.id))
-    .then(results => res.send(results.rows))
-    .catch(error => console.log(error));
```

Comment:
I would recommend using different error handling other than console logging by means of middleware

https://derickbailey.com/2014/09/06/proper-error-handling-in-expressjs-route-handlers/

I found this a good read

Reply:
Express's error handling middleware is pretty neat, but FYI no need to pass the exact error message to the frontend as the article illustrates!